src/com/company/Database.java

```diff
@@ -0,0 +1,30 @@
+package com.company;
+
+import java.sql.*;
+
+import static com.company.Main.statement;
+
+public class Database {
+    static String query = "insert into audit values(?,?,now(),?,?,?)";
+    private static String url = "jdbc:mysql://localhost:3306/elixr_intern_poc_01";
+    private static String user="vikas";
+    private static String password="root";
+
+    void databaseInsertion(String txtFilePath,String searchWord,int count,String status,String errorMessage) {
+        try {
+            Connection con = DriverManager.getConnection(url, user, password);
```

Comment:
Shouldn't the connection be closed here?

Reply:
closed the connection